src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,54 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_TYPE = 0;
+    private static final int AMOUNT = 1;
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String content = readFile(fromFileName);
+        writeFile(toFileName, content);
+    }
```

Comment:
The `getStatistic` method is doing too much. It's reading from a file, processing the data, and then writing to another file. This is a violation of the Single Responsibility Principle. It would be better to split this method into smaller methods, each with its own responsibility.